.github/workflows/release.yml

```diff
@@ -52,6 +52,13 @@ jobs:
       run: |
         VERSION=$(git describe --tags --match 'v*' --always | tr -d 'v')
         echo "VERSION=${VERSION}" >> $GITHUB_ENV
+        echo ${{ env.VERSION }}
+
+    - name: Get manifest version
+      run: |
+        MANIFEST_VERSION=$(grep version fpm.toml | head -1 | tr -d ' ' | tr -d 'version=')
+        echo "MANIFEST_VERSION=${MANIFEST_VERSION}" >> $GITHUB_ENV
+        echo ${{ env.MANIFEST_VERSION }}
```

Comment:
Using pre-processing flags, there are a few issues with the single-file-version: 

1) There are a few issues with passing manifest version with a pre-processor flag: 

1) the single-source-file version is not built with fpm, so we need to first query the manifest version into a variable,